squad/api/rest.py

```diff
@@ -480,19 +521,25 @@ def testjobs(self, request, pk=None):
         serializer = TestJobSerializer(page, many=True, context={'request': request})
         return self.get_paginated_response(serializer.data)
 
-    @detail_route(methods=['get'], suffix='email')
-    def email(self, request, pk=None):
-        """
-        This method produces the body of email notification for the build.
-        By default it uses the project settings for HTML and template.
-        These settings can be overwritten by using GET parameters:
-         * output - sets the output format (text/plan, text/html)
-         * template - sets the template used (id of existing template or
-                      "default" for default SQUAD templates)
-        """
+    def __return_delayed_report(self, request):
         output_format = request.query_params.get("output", "text/plain")
         template_id = request.query_params.get("template", None)
         baseline_id = request.query_params.get("baseline", None)
+        email_recipient = request.query_params.get("email_recipient", None)
+        callback = request.query_params.get("callback", None)
+        callback_token = request.query_params.get("callback_token", None)
+        # keep the cached reports for 1 day by default
+        data_retention_days = request.query_params.get("keep", 1)
```

Comment:
I don't see the 'keep' parameter described in the API docs. Is this intentional?

Reply:
no, mistake. Thanks for reporting. I will add it